### PR TITLE
refactor: use lowercase discord.js everywhere

### PR DESCRIPTION
**Pull request by GodderE2D**
**Please describe the changes this PR makes and why it should be merged:**

The main site ([discord.js.org](https://discord.js.org)) uses lowercase discord.js, and so does the server. This pull request changes all "Discord.js" instances to "discord.js". I have no idea if this is going to break anything as getting up a local version didn't work for me, so the deploy preview is my friend :D

Deploy Preview: https://deploy-preview-1015--discordjs-guide.netlify.app/
Update: I think everything worked! 🎉

### CODE REVIEW

File: guide/miscellaneous/parsing-mention-arguments.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Parsing mentions
 
-Discord.js is already geared to help you handle mentions using `message.mentions`.
+discord.js is already geared to help you handle mentions using `message.mentions`.
```
Comment (almostSouji) [inline guide/miscellaneous/parsing-mention-arguments.md:3]: Revert, beginning of a sentence

**Review by almostSouji — CHANGES_REQUESTED** [review]
Unsure about the other ones, cc @Danktuary , but the one I marked definitely needs to be capital, as it's the beginning of a sentence in a paragraph

**Comment by GodderE2D** [comment]
(For above commit) I'm not sure if that counts as a sentence because it is a heading, should I revert that?

**Review by almostSouji — APPROVED** [review]
No, you're right, headings, too, should be capital

**Pull request merged**